src/spec/expectations.cr

```diff
@@ -220,7 +220,14 @@ module Spec
     end
 
     def match(actual_value)
-      actual_value.includes?(@expected_value)
+      expected_value = @expected_value
+      if expected_value.is_a?(Hash) && actual_value.is_a?(Hash)
+        expected_value.all? do |key, value|
+          actual_value[key]? == value
```

Comment:
This behavior isn't consistent with how `includes?` works. I don't know what problem it's supposed to solve.

Reply:
It was discussed at length at Gitter as well as in several comments in before-mentioned issue. IMHO giving downvotes before you understand the problem at hand doesn't make much sense and if anything it shows an ill-will you hold for some reason...